app/sim_test.go

```diff
@@ -210,9 +214,12 @@ func TestAppImportExport(t *testing.T) {
 		{app.keys[ibctransfertypes.StoreKey], newApp.keys[ibctransfertypes.StoreKey], [][]byte{}},
 		// {app.keys[evmtypes.StoreKey], newApp.keys[evmtypes.StoreKey], [][]byte{}},
 		{app.keys[feemarkettypes.StoreKey], newApp.keys[feemarkettypes.StoreKey], [][]byte{}},
-		//{app.keys[inflationtypes.StoreKey], newApp.keys[inflationtypes.StoreKey], [][]byte{}},
+		{app.keys[inflationtypes.StoreKey], newApp.keys[inflationtypes.StoreKey], [][]byte{}},
 		// {app.keys[erc20types.StoreKey], newApp.keys[erc20types.StoreKey], [][]byte{}},
-		//{app.keys[epochstypes.StoreKey], newApp.keys[epochstypes.StoreKey], [][]byte{}},
+		{app.keys[epochstypes.StoreKey], newApp.keys[epochstypes.StoreKey], [][]byte{
```

Comment:
It should be commented, [Ref](https://github.com/Canto-Network/Canto/blob/main/app/sim_test.go#L218-L219)

Reply:
done in https://github.com/Canto-Network/Canto/blob/main/app/sim_test.go#L218-L219 

comment is written by myself but will use comment you linked when merge main branch.